pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1869,7 +1893,17 @@ func newRole(namespace, name string) *rbacv1.Role { //nolint: unparam
 	}
 }
 
-func newClusterResourceQuota(username, tier string) *quotav1.ClusterResourceQuota {
+func newClusterResourceQuota(t *testing.T, username, tier string) *quotav1.ClusterResourceQuota {
+	quota := corev1.ResourceQuotaSpec{}
+	quotaContent := `
+quota:
+  hard:
+    limits.cpu: 1750m
+    limits.memory: 7Gi`
+
+	err := yaml.Unmarshal([]byte(quotaContent), &quota)
```

Comment:
could we try to see what's the result of this unmarshalling so we can keep the code as it was before, but maybe with adding the missing entries in the quota specs? (in line 1919 on the updated file)

Reply:
I made a wrong assumption while debugging failures in the test. Sorry for that - after another investigation, I figured out that the problems were somewhere else.
Brought back the original way of creating it in e49bb97
Thanks a lot for raising this concern :+1:

Reply:
ok